changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.13-beta1": [
+      "[New] Add support for notifications of pull request reviews - #14175",
+      "[Fixed] Pull requests adhere to temporal laws again - #14238",
```

Comment:
Not sure if there is precedent of funny release notes like this, but I personally love it 😂